src/components/AdminPage/SideNavBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Menu, Icon } from 'semantic-ui-react';
+import { Link } from 'react-dom';
+
+function SideNavBar() {
+  return (
+    <Menu className="SideNavBar" positon="left" borderless vertical>
```

Comment:
I think I'll add an eslint rule to have attributes on separate lines, but ignore that for now.
```
  <Menu
    className="SideNavBar"
    positon="left"
    borderless
    vertical
  >
```